gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/BaseEntity.java

```diff
@@ -183,6 +261,17 @@ public B set(String name, Blob value) {
       return self();
     }
 
+    public B set(String name, Blob first, Blob second, Blob... others) {
```

Comment:
Lets add javadoc (for all cases) to make it clear that this version is setting a list of values.